test/e2e/e2e.bats

```diff
@@ -2,14 +2,17 @@
 
 source ./test/helper/helper.sh
 
-export PARAMS_SOURCE="docker://docker.io/library/busybox:latest"
-export PARAMS_DESTINATION="docker://registry.registry.svc.cluster.local:32222/busybox:latest"
-
-# setting tls-verify as false disables the HTTPS client as well, something we need for e2e testing
-export PARAMS_TLS_VERIFY="false"
+PARAMS_SOURCE=$(echo ${E2E_SC_PARAMS_SOURCE:-} | tr -d '"')
+PARAMS_DESTINATION=$(echo ${E2E_SC_PARAMS_DESTINATION:-} | tr -d '"')
+PARAMS_TLS_VERIFY=$(echo ${E2E_SC_PARAMS_TLS_VERIFY:-} | tr -d '"')
```

Comment:
```bash
declare -rx PARAMS_TLS_VERIFY="${E2E_SC_PARAMS_TLS_VERIFY:-}"
```
This is the type of declaration we are looking for, please try.

Reply:
I found the mistake of the additional double quotes that I was getting, I was assigning the parameter `"URL"` in the Makefile instead of only `URL`, a rookie mistake at my end :sweat: .